### ISSUE TITLE
wow, you guys are using cats now?

### ISSUE THREAD

**Issue by johanatan** [BoomTownROI/cats-examples#2]
Very smart move.

**Comment by rstradling** [comment]
Yes.  Boomtown is using cats.  Boomtown has been using pieces of it for probably about 9 months.

**Comment by johanatan** [comment]
Try 12 months. I wrote code for them using cats 1 year ago & was fired for "not following everyone else's [inferior] patterns".
